test/distributed/org/apache/cassandra/distributed/test/accord/AccordMigrationTest.java

```diff
@@ -570,7 +570,7 @@ public void testAccordToPaxos() throws Exception
                  assertTargetPaxosWrite(runCasNoApply, 1, paxosMigratingKeys.next(), 2, 1, 1, 1, 1);
 
                  // Repair the currently migrating range from when targets were switched, but it's not an Accord repair, this is to make sure the wrong repair type doesn't trigger progress
-                 nodetool(coordinator, "repair", "-st", upperMidToken.toString(), "-et", maxAlignedWithLocalRanges.toString());
+                 nodetool(coordinator, "repair", "-st", upperMidToken.toString(), "-et", maxAlignedWithLocalRanges.toString(), "--paxos-only");
```

Comment:
Paxos only is not sufficient here, we need to repair both Paxos and the base table for migration. We are also trying to address partially applied mutations that aren't from Paxos.